ffmsindex/types.go

```diff
@@ -68,6 +68,13 @@ type Header struct {
 	}
 	FileSize int64
 	Digest   [20]byte
+	LAVFOpts []LAVFOption
+}
+
+// LAVFOption is a single key-value pair for a LAVF option.
+type LAVFOption struct {
+	Key   []byte
+	Value []byte
```

Comment:
These should be `string`. Rest LGTM.

Reply:
I don't think there's a straightforward way to read bytes into a string though, so it would complicate parsing significantly, is that fine?

Reply:
It looked overcomplicated based on docs I'd read initially, but looks like we can just read to slices and then cast - done.